### PR TITLE
nushell: update to 0.22.0, update distfile

### PR DESCRIPTION
**Pull request by Rheydskey**
Hi,
I have updated nushell and changed the repositories of the compilation files.
Rheydskey

### CODE REVIEW

File: srcpkgs/nushell/template [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 pkgname=nushell
-version=0.21.0
+version=0.22.0
 revision=1
 # raw-cpuid does not support other archs
 # https://github.com/nushell/nushell/issues/485
 # remove when the issue is fixed (should fix all archs)
 archs="x86_64* i686*"
-wrksrc="nu-${version}"
+wrksrc="nushell-${version}"
```
Comment (ericonr) [inline srcpkgs/nushell/template:9]: That's default, you can just remove the line.

**Comment by ericonr** [comment]
Please follow commit style properly: `nushell: update to 0.22.0, update distfile`

That said, why move to github instead of continuing using crates?

**Comment by Rheydskey** [comment]
> Please follow commit style properly: `nushell: update to 0.22.0, update distfile`
> 
> That said, why move to github instead of continuing using crates?

I switched to github because using firefox I couldn't download the source archive. So I thought that with wget or curl (I don't know what xbps uses) the archive wouldn't download either.

**Comment by ericonr** [comment]
It might have changed to `nushell/nushell-$version`, but eh. Fetching from github works.

**Comment by ericonr** [comment]
You updated PR title, but I need you to update the commit message :)

You should also squash the commits, ideally we want a single commit per package.

**Comment by ericonr** [comment]
It seems you'll have to wait for rust 1.47 to be packaged, because nushell requires nightly features such as a new constant.

**Comment by Rheydskey** [comment]
> It seems you'll have to wait for rust 1.47 to be packaged, because nushell requires nightly features such as a new constant.

Yes

**Comment by ericonr** [comment]
> You updated PR title, but I need you to update the commit message :)

> You should also squash the commits, we want a single commit per package.

**Comment by Rheydskey** [comment]
> > You updated PR title, but I need you to update the commit message :)
> 
> > You should also squash the commits, we want a single commit per package.

How I do it ?

**Comment by ericonr** [comment]
https://git-rebase.io/ is one guide for it. Just use interactive rebase, mark the last commit to squash it, and fix the message.

**Comment by Rheydskey** [comment]
> https://git-rebase.io/ is one guide for it. Just use interactive rebase, mark the last commit to squash it, and fix the message. Then force push to the branch.

It's okay, I've done everything

**Comment by ericonr** [comment]
Please update the PR, new Rust has been merged.

**Comment by ericonr** [comment]
Please update to 0.24.1: https://github.com/nushell/nushell/releases

And see if you can fix the tests, I'm not sure why they fail.

Ping me when done, and a happy new year

**Comment by Rheydskey** [comment]
I made a bad manipulation

**Pull request closed without merging**